modules/appeals_api/spec/requests/supplemental_claims/v0/supplemental_claims/evidence_submissions_controller_spec.rb

```diff
@@ -26,7 +25,9 @@ def stub_upload_location(expected_location = 'http://some.fakesite.com/path/uuid
       success_status: :accepted
     ) do
       def make_request(auth_header)
-        post(path, params:, headers: headers.merge(auth_header))
+        post(path,
+             params:, # TODO: read SSN from body instead:
+             headers: auth_header.merge({ 'X-VA-SSN' => supplemental_claim.veteran.ssn }))
```

Comment:
This is a temporary band-aid until I can make a subsequent PR to update the `evidence-submissions` endpoint for supplemental claims. It sill expects an `X-VA-SSN` header instead of a JSON body. Previously, this test used the header fixtures intended for the `create` endpoint to set that header, but now that those fixtures no longer have an SSN, I'm setting it manually here for now.